backend/DB.ts

```diff
@@ -1,24 +1,28 @@
 import Umzug from "umzug";
-import s from "umzug/lib/storages/SequelizeStorage.js";
+import s from "sequelize";
+import ss from "umzug/lib/storages/SequelizeStorage.js";
 import { router } from "./Router.js";
 // @ts-ignore
 import db from "../models/index.cjs";
 
-const { default: SequelizeStorage } = s;
+const { default: SequelizeStorage } = ss;
+const { Sequelize } = s;
 
 export const sequelize = db.sequelize;
 
 export const dbReady = new Promise<void>((resolve, reject) => {
   const intervalId = setInterval(() => {
     console.log("Attempting to connect to db");
     sequelize.authenticate().then(
-      () => {
+      async () => {
```

Comment:
Why is this async? I don't see await being used?

Reply:
oh yeah it doesn't need to be, just a leftover from some debugging i was doing - will remove